examples/example-menubar/src/index.ts

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { CommandRegistry } from '@lumino/commands';
+import { Menu, MenuBar, PanelLayout, Widget } from '@lumino/widgets';
+
+import '../style/index.css';
+
+/**
+ * Wrapper widget containing the example application.
+ */
+class Application extends Widget {
+  constructor() {
+    super({ tag: 'main' });
+  }
+}
+
+/**
+ * Skip link to jump to the main content.
+ */
+class SkipLink extends Widget {
+  /**
+   * Create a HTMLElement that statically links to "#content".
+   */
+  static createNode(): HTMLElement {
+    const node = document.createElement('a');
+    node.setAttribute('href', '#content');
+    node.innerHTML = 'Skip to the main content';
+    node.classList.add('lm-example-skip-link');
+    return node;
+  }
+
+  constructor() {
+    super({ node: SkipLink.createNode() });
+  }
+}
+
+/**
+ * A Widget containing some content to provide context example.
+ */
+class Article extends Widget {
+  /**
+   * Create the content structure.
+   */
+  static createNode(): HTMLElement {
+    const node = document.createElement('div');
+    node.setAttribute('id', 'content');
+    node.setAttribute('tabindex', '-1');
+    const h1 = document.createElement('h1');
+    h1.innerHTML = 'MenuBar Example';
+    node.appendChild(h1);
+    const label = document.createElement('label');
+    label.appendChild(
+      document.createTextNode('A textarea to demonstrate the tab handling.')
```

Comment:
I'll add a comment. It is basically there so that there is some content that can be tabbed to. Otherwise the tab out of the menubar takes you into the browser UI and that can often be a bit harder to see where it has gone (I guess an a11y issue in itself).